tools/restatectl/src/commands/log/describe_log.rs

```diff
@@ -62,11 +62,18 @@ async fn describe_log(connection: &ConnectionInfo, opts: &DescribeLogIdOpts) ->
     c_println!("{}", table);
 
     let mut chain_table = Table::new_styled();
-    chain_table.set_styled_header(vec!["SEGMENT", "STATE", "BASE LSN", "TAIL LSN", "KIND"]);
+    chain_table.set_styled_header(vec![
+        "SEGMENT",
+        "STATE",
+        "SEGMENT START",
+        "SEGMENT TAIL",
```

Comment:
I think using the "tail" here is misleading and can be confusing to operators. In fact, we only need to show the starting point of the segment (it's inclusive). Can we remove the tail from here? 

Maybe just "FROM" enough (I would try and use a single word to avoid the space-related confusion).

My suggestion:
- IDX
- KIND
- FROM
- REPLICATION [only gets a value if this is replicated loglet, prints ReplicationProperty]
- SEQUENCER [only gets a value if this is replicated loglet]
- E-NODESET (EffectiveNodeSet)